src/ast.js

```diff
@@ -247,7 +247,7 @@ var operatorMap = {};
     operatorMap[TOK.T_PERCENT] = Sk.astnodes.Mod;
 }());
 
-function py3_operators(py3) {
+function setupOperators(py3) {
```

Comment:
Shouldn't this be:
```
Sk.ast.setupOperators = function py3_operators(py3) {
...
}
Sk.exportSymbol("Sk.ast.setupOperators", Sk.ast.setupOperators);
```

Everywhere else I can see that we use `Sk.exportSymbol()`, the thing being exported is already set up with its fully-qualified name.